<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_sfinfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_stats">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sfinfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SF_INFO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stats">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stats</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>filename</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>211</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Effects</string>
    </property>
    <widget class="QRadioButton" name="radioButton_removedc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove DC</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_normalize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Normalize</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_reverse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_echo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Echo</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_changespeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Change speed</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_amplify">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Amplify</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_randomsnips">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Random snips</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>421</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="removedc">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove DC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="normalize">
     <widget class="QLabel" name="label_dBnormalize">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>59</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>dB</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_normalize">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>110</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-48.000000000000000</double>
      </property>
      <property name="maximum">
       <double>0.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Normalize to:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Normalize</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="amplify">
     <widget class="QSlider" name="verticalSlider_amplify">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>22</width>
        <height>201</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplify</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_amplify">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reverse">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reverse</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="echo">
     <widget class="QSlider" name="horizontalSlider_echonum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_echodamp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_echonum">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of echos</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_echodelay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_echodelay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay milliseconds</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_echodamp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplitude damping reduction %</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Echo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_echonumval">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_echodelayval">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_echodampval">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="changespeed">
     <widget class="QSlider" name="horizontalSlider_speedChange">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Slow</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fast</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Change speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_speedVal">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Random Snips</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_maxlen">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>650</number>
      </property>
      <property name="value">
       <number>400</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_reps">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_maxlenvalue">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>400</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_repsvalue">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_maxlen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Length</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_reps">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Snip Reps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_seconds">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Seconds:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_secondsval">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Play">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>420</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
